spark/dl/src/main/scala/com/intel/analytics/bigdl/nn/Sequential.scala

```diff
@@ -30,61 +30,91 @@ import scala.reflect.ClassTag
 class Sequential[T: ClassTag]
 (implicit ev: TensorNumeric[T]) extends Container[Activity, Activity, T] {
 
+  private var evaluateOnly = false
+
+  /**
+   * If evaluateOnly = true, then this sequential only works in evaluate mode, and not work in
+   * training mode
+   * @param status contral evaluateOnly
+   * @return this
+   */
+  def setEvaluateOnly(status: Boolean): this.type = {
+    evaluateOnly = true
+    this
+  }
+
   override def updateOutput(input: Activity): Activity = {
-    var i = 0
-    var result = input.asInstanceOf[Activity]
-    while (i < modules.length) {
-      result = modules(i).forward(result)
-      i += 1
-    }
+    if (!evaluateOnly || !isTraining()) {
```

Comment:
If evaluateOnly = true, then this sequential only works in evaluate mode

if(!evaluateOnly || isTraining)???